mteb/abstasks/AbsTaskInstructionRetrieval.py

```diff
@@ -513,21 +512,21 @@ def evaluate(self, model, split="test", **kwargs):
                     None,
                     **kwargs,
                 )
-                overall_changed_scores["length_ablation"] = {
+                overall_changed_scores[lang]["length_ablation"] = {
                     "keywords": scores_w_keywords,
                     "short_instructions": scores_w_short_instr,
                     "base": scores_base,
                 }
 
         return overall_changed_scores
 
-    def _evaluate_monolingual(
+    def _evaluate_subset(
         self,
         retriever: InstructionRetrievalEvaluator,
         corpus: Dict[str, Dict[str, str]],
         queries: Dict[str, str],
         relevant_docs: Dict[str, Dict[str, int]],
-        qrels_name: str,
+        qrels_name: str,  # this is currently unused (potential bug?)
```

Comment:
@orionw can I ask you to look at this?

Reply:
This is correct, it was an older version that can be removed since it is unused.  Thanks for catching!